src/components/HowSoloStakingWorks.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
```

Comment:
I would rename this file to keep it consistent with the rest of the Staking components.

Should be `StakingHowSoloWorks.js`.

Note: another thing to think of is creating a folder `components/staking` to group all the staking specific components together.